samples/csharp/MicrosoftEduGraphSamples/Workflows/SubmissionWorkflow.cs

```diff
@@ -168,5 +201,109 @@ public async Task BatchRequestWorkflow()
+            var assignmentId = assignment.Id;
+            Console.WriteLine($"Assignment created {assignmentId}");
+
+            await Assignment.SetUpAssignmentFeedbackResourcesFolderAsync(graphClient, _config["classId"], assignmentId);
+            Console.WriteLine("SetupResourceFolder creation successful");
+
+            // Check feedback resource folder
+            int retryNum = 0;
+            while (assignment.FeedbackResourcesFolderUrl == null && retryNum <= MAX_RETRIES)
+            {
+                assignment = await Assignment.GetAssignmentAsync(graphClient, _config["classId"], assignmentId);
+                retryNum++;
+            }
+            Console.WriteLine($"Feedback resources folder: {assignment.FeedbackResourcesFolderUrl}");
+
+            //Publish assignment
+            assignment = await GlobalMethods.PublishAssignmentsAsync(graphClient, assignment.Id);
+            Thread.Sleep(7000);
+
+            //Get assignment submissions
+            var submissions = await Submission.GetSubmissionsAsync(
+                graphClient,
+                _config["classId"],
+                assignmentId);
+            var submissionId = submissions.Value[0].Id;
+
+            // Create a new submission feedback resource
+            var feedbackResource = await Submission.CreateFeedbackResourceOutcomeAsync(
+                graphClient,
+                _config["classId"],
+                assignmentId,
+                submissionId);
+            Thread.Sleep(2000);
+            Console.WriteLine($"Feedback resource created: {feedbackResource.Id}");
+
+            // Get submission outcomes
+            var submissionOutcomes = await Submission.GetSubmissionOutcomesAsync(
+                graphClient,
+                _config["classId"],
+                assignmentId,
+                submissionId);
+
+            // Take the points outcome id
+            var pointsOutcomeId = submissionOutcomes.Value.Where(x => x.OdataType == "#microsoft.graph.educationPointsOutcome").Select(x => x.Id).FirstOrDefault();
+
+            // Create the points outcome body
+            var pointsOutcome = new EducationPointsOutcome
+            {
+                OdataType = "#microsoft.graph.educationPointsOutcome",
+                Points = new EducationAssignmentPointsGrade
+                {
+                    OdataType = "#microsoft.graph.educationAssignmentPointsGrade",
+                    Points = 90
+                }
+            };
+
+            // Update the submission points outcome
+            var returned = await Submission.PatchOutcomeAsync(
+                graphClient,
+                _config["classId"],
+                assignmentId,
+                submissionId,
+                pointsOutcomeId,
+                pointsOutcome);
+            Thread.Sleep(2000);
+            Console.WriteLine($"Points outcome updated: {pointsOutcome.Points.Points}");
+
+            // Refresh list of submission outcomes
```

Comment:
this can be omitted in my opinion.

Reply:
Initially after creating feedback resource and refreshing the list of outcomes to see the newly created feedback resource.
Hence this part of code is needed.